### PR TITLE
LPD-35067 | master

### PR DESCRIPTION
**Pull request by marcosapmf**
Jira ticket: https://liferay.atlassian.net/browse/LPD-35067

### CODE REVIEW

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:stable

File: modules/dxp/apps/analytics/analytics-reports-rest-impl/src/main/java/com/liferay/analytics/reports/rest/internal/resource/v1_0/AssetMetricResourceImpl.java [hunk 1]
```diff
@@ -32,22 +41,63 @@ public AssetMetric getGroupAssetMetric(
...
+	}
+
+	private List<Long> _getAnalyticsCloudChannelIds(
+			List<DepotEntryGroupRel> depotEntryGroupRels)
+		throws Exception {
+
+		List<Long> analyticsCloudChannelIds = new ArrayList<>();
+
+		for (DepotEntryGroupRel depotEntryGroupRel : depotEntryGroupRels) {
+			Group group = _groupService.getGroup(
```
Comment (interaminense) [inline modules/dxp/apps/analytics/analytics-reports-rest-impl/src/main/java/com/liferay/analytics/reports/rest/internal/resource/v1_0/AssetMetricResourceImpl.java:79]: Hey @marcosapmf, I tried to deploy it locally and got an error. Shouldn't it be:
```Group group = _groupLocalService.getGroup(```

**Comment by marcosapmf** [comment]
resent to https://github.com/liferay-ac/liferay-portal/pull/1821

File: modules/dxp/apps/analytics/analytics-reports-rest-impl/src/main/java/com/liferay/analytics/reports/rest/internal/resource/v1_0/AssetMetricResourceImpl.java (hunk 1 above)
Comment (marcosapmf) [reply modules/dxp/apps/analytics/analytics-reports-rest-impl/src/main/java/com/liferay/analytics/reports/rest/internal/resource/v1_0/AssetMetricResourceImpl.java:79]: Oops... thanks for catching this, @interaminense :smile:

**Pull request closed without merging**